apps/judicial-system/web/src/utils/hooks/useEventLog/index.ts

```diff
@@ -29,6 +30,8 @@ const useEventLog = () => {
     [createEventLogMutation, formatMessage],
   )
 
-  return { createEventLog }
+  return {
+    createEventLog: { action: createEventLog, loading: createEventLogLoading },
+  }
```

Comment:
This is a new pattern. Previously we have returned a flat object. Please consider using the same pattern everywhere.